Lib/fontTools/cffLib/__init__.py

```diff
@@ -256,7 +256,8 @@ def convertCFFToCFF2(self, otFont):
 				charStrings.charStringsIndex.fdArray = fdArray
 			else:
 				charStrings.fdArray = fdArray
-			fontDict = FontDict(isCFF2=True)
+			fontDict = FontDict()
+			fontDict.setCFF2(True)
```

Comment:
(i preferred the `isCFF=True` in the constructor, but nevermind)

Reply:
Me too, but there is an assert at the start of BaseDict.__init__() which prevents this. I don't understand why the assert exists, and so didn't want to remove it. I can't see that the assert serves a useful check. If you agree, I would like to remove it.